### PR TITLE
Player wall

### PR DESCRIPTION
**Pull request by DylanHul**
Merging this branch adds:
-an Enemy component (one only)
-rendering of enemy

### CODE REVIEW

File: client/components/board.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ import BoardElement from './boardElement'
 class Board extends React.Component {
 
   render(){
-    const {tileGrid, playerPosition} = this.props
+    const {tileGrid, playerPosition, enemyPosition} = this.props
+    console.log("this is enemyPosition from board: ", enemyPosition);
```
Comment (hudson-prestidge) [inline client/components/board.js]: !!! 🍍  🐹

File: client/components/enemy.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class Enemy extends React.Component {
+
+  constructor(props){
+    super(props)
+
+
+  }
+
+  render(){
+      console.log("here is the enemy", this.props);
```
Comment (hudson-prestidge) [inline client/components/enemy.js]: 🎅  console logs

File: client/connectors/enemyConnector.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import Enemy from '../components/enemy'
+
+const mapStateToProps = (state) => {
+  return {
+    x: state.enemies.enemy.position.x,
+    y: state.enemies.enemy.position.y
```
Comment (DylanHul) [inline client/connectors/enemyConnector.js:8]: Correct indenting please 🐸  👘  ✌️

File: client/components/game.js [hunk 4]
```diff
@@ -22,7 +23,7 @@ class Game extends React.Component {
           break;
         case('ArrowDown'):
           this.props.playerMoveDown()
-          break;
+        break;
```
Comment (hudson-prestidge) [inline client/components/game.js]: indenting 🎽

**Comment by DylanHul** [comment]
👡

**Comment by DylanHul** [comment]
💴

**Comment by DylanHul** [comment]
🍡

**Pull request merged**